### PR TITLE
WIP: Make the ci pass

### PR DESCRIPTION
**Pull request by St0rmingBr4in**
Some changes to make the CI pass. Still WIP
There's changes needed in the way docker-compose is used to run the tests.

### CODE REVIEW

File: src/authentication.ts [hunk 1]
```diff
@@ -20,14 +20,14 @@ export function BasicAuth(app: Express.Application) {
...
-  app.use((req: Request, res, next) => {
+  app.use((req: CustomRequest, res, next) => {
     // If user already has user account details, it means it has already been authenticated
     // and we don't need to do it again. We rather skip to the next middleware.
-    if (req.session.user) {
-      req.user = req.session.user;
-      next();
-      return;
-    }
+    // if (req.session.user) {
```
Comment (kamaradclimber) [inline src/authentication.ts:26]: are we sure we want to comment this code?

Comment (Lqp1) [reply src/authentication.ts:26]: I'm working on it, but I'm fighting with cryptic ts errors :(

Comment (St0rmingBr4in) [reply src/authentication.ts:26]: No we don't want to, @Lqp1 is working on this

Comment (Lqp1) [reply src/authentication.ts]: Fixed!

**Comment by kamaradclimber** [comment]
This PR could likely be closed

**Comment by kamaradclimber** [comment]
Since the CI has been fixed, I'm closing the PR, feel free to reopen if necessary

**Pull request closed without merging**